src/shogun/machine/gp/GaussianLikelihood.h

```diff
@@ -63,8 +64,9 @@ class CGaussianLikelihood: public CLikelihoodModel
 	 */
 	void set_sigma(float64_t sigma)
 	{
-		REQUIRE(sigma>0.0, "Standard deviation must be greater than zero\n")
-		m_sigma=sigma;
+		REQUIRE(sigma>0.0, "Standard deviation (%f) must be greater than zero\n",
```

Comment:
OK!

Uniform is good. Feel free to change things that you dont like on the fly